pkg/csset/csset_test.go

```diff
@@ -29,7 +30,9 @@ func TestCSSetTwo(t *testing.T) {
 	set := NewCSSet("")
 	set.Add("one")
 	set.Add("two")
-	assert.Equal(t, "one,two", set.String())
+	exp := [2]string{"one", "two"}
```

Comment:
If we end up using csset anywhere else, maybe we should write a test helper for this. Something that blends with the testify/assert package would be nice.